src/rs/src/text.rs

```diff
@@ -215,18 +215,18 @@ impl<'a> SkillThing<'a> {
 				s.push_str("|1 Phantom");
 				Cow::from(s)
 			}
-			Skill::bubbleclear => Cow::from("Remove statuses (positive & negative) from target creature or permanent, & heal target creature 1.\nTarget gains a bubble. Bubbles nullify the next spell, ability, or spell damage used by opponent that targets or damages affected card"),
+			Skill::bubbleclear => Cow::from("Strip target creature or permanent of all statuses. If target is a creature, heal it by 1.\nTarget gains Bubble, protecting it from 1 spell or ability"),
```

Comment:
can you provide an example (Is Brawl not a spell)?

Reply:
Brawl is a spell, bubble would protect if brawl caused psion to attack bubble creature, but otherwise bubble wouldn't be useful there

Reply:
Got it - AOE spells are not propagated to children.

Reply:
Remove all statuses from target creature or permanent. If target is a creature, heal it by 1 HP.\nTarget gains Bubble, 
protecting it from the next time it is directly targeted.

Reply:
`protecting it from the next time it receives spell damage or is targeted`

Reply:
Brawl really is an odd duck...everything else uses masscc(). Revised above comment as suggested. Will modify and PR - thanks for help. This is a large part of why I keep coming back to this file - to better learn business logic behind keywords (spells, vs abilities, vs abilities which are spells which call existing spell functions, vs abilities which are spells which get classified as spells)

Reply:
bubble doesn't protect from masscc due to targeting, but because most masscc is spell damage. Pandemonium snipe for example will bypass bubble

Reply:
hmm. more complicated due to subfunctions:

`protectonce` and `protectoncedmg` are triggered.

`protectonce` resolves only if it's not selftarget, so nothing should happen to bubble in this case (get buff, don't lose bubble).

`protectoncedmg` however doesn't have the selftarget safeguard, so will remove the bubble due to the fact that something like `` modifies HP via the **dmg() call**.

**so we need to add `is targeted *by opponent*`**
Also, reordered tail to add comma for clarity.

`Remove all statuses from target creature or permanent. If target is a creature, heal it by 1 HP.\nTarget gains Bubble, protecting it from the next time it is targeted by opponent, or receives spell damage from any source.`

e.g.:
protectonce: `rage pot` is a `targeting spell`, but is `prevented from triggering bubble due to self-cast`.
protectoncedmg: creature targeted by `rage pot` takes dmg (due to the mechanic of how hp is being handled...dmg() is called), so it `removes bubble`.

This phrasing should work for `Brawl`, since the creature with bubble isn't directly targeted, and damage source is from a creature (as opposed to directly from the spell card).